src/core/File.ts

```diff
@@ -117,6 +117,11 @@ export class File {
 
     public groupHandler: Plugin;
 
+    /**
+     * Flag to indicate to fusebox whether or not contents has been set to consumable javascript
+     */
+    public transpiled: Boolean = false;
```

Comment:
Yeah, I can do that. Thanks. I mostly made this to illustrate my goal. I'll refine it more once I validate it is a valid approach